app/models/manageiq/providers/container_manager.rb

```diff
@@ -20,7 +20,21 @@ class ContainerManager < BaseManager
     has_many :container_build_pods, :foreign_key => :ems_id, :dependent => :destroy
     has_many :container_templates, :foreign_key => :ems_id, :dependent => :destroy
     has_one :container_deployment, :foreign_key => :deployed_ems_id, :inverse_of => :deployed_ems
+
+    # Shortcuts to chained joins, mostly used by inventory refresh.
     has_many :computer_systems, :through => :container_nodes
+    has_many :computer_system_hardwares, :through => :computer_systems, :source => :hardware
+    has_many :computer_system_operating_systems, :through => :computer_systems, :source => :operating_system
+    has_many :container_volumes, :through => :container_groups
+    has_many :container_definitions, :through => :container_groups
+    has_many :container_port_configs, :through => :container_definitions
+    has_many :container_env_vars, :through => :container_definitions
+    has_many :security_contexts, :through => :container_definitions
+    has_many :container_service_port_configs, :through => :container_services
+    has_many :container_routes, :through => :container_services
```

Comment:
hum, yeah, we should align with https://github.com/Ladas/manageiq/blob/46ec716c8fc715d0847d63d3d4c9a35135e21f01/db/migrate/20170530102630_clean_up_duplicates_in_containers_tables.rb#L61

we should use :ems_id where we can, the queries will be quicker